src/Browser.elm

```diff
@@ -1,9 +1,12 @@
-module Browser exposing
-    ( sandbox
-    , element
-    , document, Document
-    , application, UrlRequest(..)
-    )
+module Browser
```

Comment:
These changes were made automatically by `elm-format` in my text editor. Leaving them in this pull request assuming that the `elm-format` formatting is OK. Please let me know if it should be removed.